Iterators and Closures/Standard Library Types/Arc/task.md

```diff
@@ -12,3 +12,7 @@ Check out the chapter [Shared-State Concurrency](https://doc.rust-lang.org/book/
 
   `child_numbers` should be a clone of the Arc of the numbers instead of a thread-local copy of the numbers.
 </div>
+
+<div class="hint">This is a simple exercise if you understand the underlying concepts, but if this
+is too much of a struggle, consider reading through all of <a href="https://doc.rust-lang.org/stable/book/ch16-00-concurrency.html">Chapter 16</a> in the Book.
```

Comment:
Maybe we can reffer not only to the book chapter, but to  the following section we have inside the course as well? Like "consider reading or return after completing the next section"?